crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -99,7 +99,13 @@ fn fragment(in: FragmentInput) -> @location(0) vec4<f32> {
         output_color = tone_mapping(output_color);
 #endif
 #ifdef DEBAND_DITHER
-        output_color = dither(output_color, in.frag_coord.xy);
+    var output_rgb = output_color.rgb;
+    output_rgb = pow(output_rgb, vec3<f32>(1.0 / 2.2));
+    output_rgb = output_rgb + screen_space_dither(in.frag_coord.xy);
+    // This conversion back to linear space is required because our output texture format is
+    // SRGB; the GPU will assume our output is linear and will apply an SRGB conversion.
+    output_rgb = pow(output_rgb, vec3<f32>(2.2));
```

Comment:
Quick (possibly stupid) question: could this back-and-forth conversion be avoided by choosing a different output texture format?

(i don't really know about how the output texture format is chosen in the first place, if srgb is a requirement then i guess that is not an option)

Reply:
Nope, great question. The answer seems to be "no" for a variety of reasons I don't fully understand. That would be my preference for sure though.